### PR TITLE
Convert modular recipes to new materials definition

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

Removes hardcoded mineral values from recipes, as these are now a macro and the values have changed.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
</details>

## Changelog

:cl: LT3
code: Modular recipes now use material definitions
/:cl:

### CODE REVIEW

File: code/__DEFINES/construction.dm [hunk 1]
```diff
@@ -74,12 +74,12 @@
 #define NARSIE_WINDOW_COLOUR "#7D1919"
 
 // Defines related to the custom materials used on objects.
-///The amount of materials you get from a sheet of mineral like iron/diamond/glass etc. 2000 Units.
-#define SHEET_MATERIAL_AMOUNT 2000
-///The amount of materials you get from half a sheet. Used in standard object quantities. 1000 units.
+///The amount of materials you get from a sheet of mineral like iron/diamond/glass etc. 100 Units.
+#define SHEET_MATERIAL_AMOUNT 100
```
Comment (Paxilmaniac) [inline code/__DEFINES/construction.dm:78]: why??

File: code/__DEFINES/construction.dm [hunk 2]
```diff
@@ -74,12 +74,12 @@
...
+///The amount of materials you get from half a sheet. Used in standard object quantities. 50 units.
 #define HALF_SHEET_MATERIAL_AMOUNT (SHEET_MATERIAL_AMOUNT/2)
-///The amount of materials used in the smallest of objects, like pens and screwdrivers. 100 units.
-#define SMALL_MATERIAL_AMOUNT (HALF_SHEET_MATERIAL_AMOUNT/10)
+///The amount of materials used in the smallest of objects, like pens and screwdrivers. 10 units.
+#define SMALL_MATERIAL_AMOUNT (HALF_SHEET_MATERIAL_AMOUNT/5)
```
Comment (Paxilmaniac) [inline code/__DEFINES/construction.dm:82]: are you touching any of this??

File: tools/silo_grapher/silo_graph_script.py [hunk 3]
```diff
@@ -0,0 +1,127 @@
+# Silo graphing tool by ArcaneMusic.
```
Comment (Paxilmaniac) [inline tools/silo_grapher/silo_graph_script.py:1]: ????

File: code/__DEFINES/construction.dm (hunk 1 above)
Comment (lessthnthree) [reply code/__DEFINES/construction.dm:78]: https://github.com/Skyrat-SS13/Skyrat-tg/pull/20916

File: code/__DEFINES/construction.dm (hunk 2 above)
Comment (lessthnthree) [reply code/__DEFINES/construction.dm:82]: https://github.com/Skyrat-SS13/Skyrat-tg/pull/20916

File: code/__DEFINES/construction.dm (hunk 1 above)
Comment (Paxilmaniac) [reply code/__DEFINES/construction.dm:78]: that doesn't explain why you're touching a core tg file?

Comment (lessthnthree) [reply code/__DEFINES/construction.dm:78]: ye I linked the wrong PR. looking, TG changed the sheet material amount

Comment (lessthnthree) [reply code/__DEFINES/construction.dm:78]: whoops these are supposed to go on https://github.com/Skyrat-SS13/Skyrat-tg/pull/21784

**Pull request closed without merging**